src/test/suite/completion.test.ts

```diff
@@ -104,9 +104,17 @@ describe("Should do completion", () => {
     await triggerSelectionAppetence();
 
     assertTextIsCommitted("console.log");
+
+    await makeAndAssertFollowingChange();
```

Comment:
I don't quite understand how is this testing the behavior relevant to this pr

Reply:
it is making an additional change after the inline suggestion was accepted, and asserting the accepted text + current change are existing. the issue was that if you type something after accepting an inline suggestion, the last change would replace the selected inline suggestion.

Reply:
So I'd extract it to another test with a proper name (and maybe comments inside the test to explain how it is testing this behavior)

Testing multiple things in one test is generally not a good idea, as it makes it harder to find what went wrong if the test fails

Reply:
Also, this sounds a bit vicarious. Isn't there a way to get the current selected text from the editor and assert that it is empty / null?

Reply:
agree, will do